docker-compose.yml

```diff
@@ -130,6 +130,15 @@ services:
       - "9200:9200"
       - "9300:9300"
 
+  # https://www.docker.elastic.co/
+  elasticsearch7:
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.1.0
+    environment:
+      - discovery.type=single-node
+    ports:
+      - "9201:9200"
+      - "9301:9300"
```

Comment:
This exposes ports 9201 and 9301 for now. When we finish migrating, we'll nix the elasticsearch container, rename this one to elasticsearch, and change the ports to the regular ones again.